internal/pkg/scaffold/go_mod.go

```diff
@@ -0,0 +1,115 @@
+	github.com/go-logr/logr v0.1.0 // indirect
+	github.com/go-logr/zapr v0.1.0 // indirect
+	github.com/go-openapi/spec v0.18.0 // indirect
+	github.com/golang/groupcache v0.0.0-20180924190550-6f2cf27854a4 // indirect
+	github.com/google/btree v0.0.0-20180813153112-4030bb1f1f0c // indirect
+	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
+	github.com/google/uuid v1.0.0 // indirect
+	github.com/googleapis/gnostic v0.2.0 // indirect
+	github.com/gophercloud/gophercloud v0.0.0-20190318015731-ff9851476e98 // indirect
+	github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.8.5 // indirect
+	github.com/imdario/mergo v0.3.6 // indirect
+	github.com/json-iterator/go v1.1.5 // indirect
+	github.com/markbates/inflect v1.0.4 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
+	github.com/operator-framework/operator-sdk v0.7.1-0.20190423132450-ec538b5b4e4c
+	github.com/pborman/uuid v0.0.0-20180906182336-adf5a7427709 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/spf13/afero v1.2.2 // indirect
+	github.com/spf13/pflag v1.0.3
+	go.opencensus.io v0.19.2 // indirect
+	go.uber.org/atomic v1.3.2 // indirect
+	go.uber.org/multierr v1.1.0 // indirect
+	go.uber.org/zap v1.9.1 // indirect
+	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2 // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	k8s.io/api v0.0.0-20181126151915-b503174bad59 // indirect
+	k8s.io/apiextensions-apiserver v0.0.0-20181126155829-0cd23ebeb688 // indirect
+	k8s.io/apimachinery v0.0.0-20181126123746-eddba98df674
+	k8s.io/client-go v2.0.0-alpha.0.0.20181126152608-d082d5923d3c+incompatible
+	k8s.io/code-generator v0.0.0-20180823001027-3dcf91f64f63
+	k8s.io/gengo v0.0.0-20181113154421-fd15ee9cc2f7
+	k8s.io/klog v0.1.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20180711000925-0cf8f7e6ed1d
+	sigs.k8s.io/controller-runtime v0.1.10
+	sigs.k8s.io/controller-tools v0.1.10
+	sigs.k8s.io/testing_frameworks v0.1.0 // indirect
+	sigs.k8s.io/yaml v1.1.0 // indirect
+)
+
+// Pinned to kubernetes-1.13.1
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20181213150558-05914d821849
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20181213153335-0fe22c71c476
+	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20181127025237-2b1284ed4c93
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20181213151034-8d9ed539ba31
+)
+
+replace (
+	github.com/coreos/prometheus-operator => github.com/coreos/prometheus-operator v0.29.0
+	k8s.io/code-generator => k8s.io/code-generator v0.0.0-20181117043124-c2090bec4d9b
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20180711000925-0cf8f7e6ed1d
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.1.10
+	sigs.k8s.io/controller-tools => sigs.k8s.io/controller-tools v0.1.8
```

Comment:
I can't recall if we have a particular reason for this to be at `v0.1.8` or we just forgot to update it.
For the SDK binary itself we're using `v0.1.10`.
https://github.com/operator-framework/operator-sdk/blob/master/Gopkg.lock#L1713

Moreover, do we even need to pin this for the user's operator project? I can't see where this is a direct dependency for the operator project.

Reply:
Originally there was a dependency issue because something in `operator-sdk/pkg` required `controller-tools`'s CRD generator, but I believe that was resolved in [this commit](https://github.com/operator-framework/operator-sdk/commit/52ad33a4bda0f78bdc8acdc49586a670fc755de5). I'll remove it here.

Reply:
Seems this is still an issue, now because `master` uses changes from an unreleased version of `controller-tools`: https://travis-ci.org/operator-framework/operator-sdk/jobs/527477327#L1019